pkg/scheduler/algorithm/predicates/predicates.go

```diff
@@ -713,8 +714,34 @@ func PodFitsResources(pod *v1.Pod, meta algorithm.PredicateMetadata, nodeInfo *s
 	}
 
 	var podRequest *schedulercache.Resource
+	// No extended resources should be ignored by default unless specified via
+	// extender configuration.
+	ignoredExtendedResources := sets.NewString()
+	// Missing extended resources will be ignored by default unless a predicate
+	// metadata is provided.
+	//
+	// This predicate is used by scheduler, kubelet, cluster autoscaler and
+	// daemonset controller.
+	//   - In scheduler, we pass the predicate metadata with the flag set to
+	//     false to this function, so scheduler will not ignore missing
+	//     extended resources.
+	//   - In kubelet/cluster autoscaler/daemonset controller, we pass nil
+	//     predicate metadata to this function, so these component will ignore
+	//     missing extended resources.
+	//
+	// To support cluster-level resources, we don't want kubelet/cluster
+	// autoscaler/daemonset controller to reject pods requesting missing
+	// extended resources, which includes cluster-level resources, so we need
+	// to set the default value of this flag to true. This is to fix the issue
+	// https://github.com/kubernetes/kubernetes/issues/53616 before we have
```

Comment:
I thought for cluster autoscaler, we do NOT want to ignore missing extended resources. Otherwise, it may cause problem with gpu autoscaling as a template node created from non-gpu nodepool would also pass this predicate checking.

Reply:
Oh, if we don't have the GPU resources in such node template, we will have to address the issue in https://github.com/kubernetes/kubernetes/pull/60332#discussion_r170441667 in order to support defaulting to `false`.